packet/endpoints.go

```diff
@@ -32,6 +33,50 @@ type ComponentsResponse struct {
 	Components []Component `json:"components"`
 }
 
+// GetWorkflowsFromTink fetches the list of workflows from tink
+func (c *Client) GetWorkflowsFromTink(hwID string) (*tw.WorkflowContextList, error) {
+
+	var result *tw.WorkflowContextList
```

Comment:
this should be defined closer to where its *actually* used and returned, not up here. All the error cases should `return nil, err...`

Reply:
i will remove this, but ideally all variable in a method should be defined in a single place and preferably onto the top.

Reply:
have placed the variables in the method signature